internal/service/audit_service_unit_test.go

```
@@ -206,6 +208,51 @@ func TestAuditAppendRedactsPayloadsAndWritesInsert(t *testing.T) {
+		t.Run(test.name, func(t *testing.T) {
+			svc, mock, cleanup := newMockAuditService(t)
+			defer cleanup()
+
+			mock.ExpectQuery("SELECT memory_space_id::text\\s+FROM credentials\\s+WHERE id = \\$1 AND team_id = \\$2").
+				WithArgs(credentialID, teamID).
+				WillReturnRows(test.rows)
+			args := make([]driver.Value, 13)
+			for index := range args {
+				args[index] = sqlmock.AnyArg()
+			}
+			args = append(args, test.memorySpaceArg)
```

golangci-lint (2.12.2)

[error] append to slice `args` with non-zero initialized length

(makezero)